### ISSUE TITLE
ppt=xyz

### ISSUE THREAD

**Issue by di-shi** [neustar/tcs-apis#165]
The test bed accepts ppt=xyz.

Test on 1/24/2023.

**Comment by kpolitz** [comment]
Will confirm, but current behavior seems to treat an unknown, unsupported ppt as "null".  Signing will generate a STIR PASSporT and verification will validate.  The range of ppt to actually be supported in Table V.2.5.2-1 is not specified.  What behavior would you suggest in this edge case, especially if we intend to add other PASSporT types.

**Comment by di-shi** [comment]
I agree for behavior that is not explicitly defined the implementation can do anything. Just test a check point. I would prefer rejecting the request with unknown ppt (with ppt is fine as SHAKEN).

**Comment by kpolitz** [comment]
Would appreciate others feedback here as well, but don't believe we want to be too restrictive of a server implementation for either an unknown "ppt" or perhaps one that a particular implementation may already support but is not yet, or will never make the 3GPP specification, e.g., "rcd".  Please let us know if we can close this Issue.  Thanks.
